app/app.js

```diff
@@ -50,6 +50,7 @@ var PatientNew = require('./pages/patientnew');
 var PatientData = require('./pages/patientdata');
 var RequestPasswordReset = require('./pages/passwordreset/request');
 var ConfirmPasswordReset = require('./pages/passwordreset/confirm');
+var SignupVerification = require('./pages/signupverification');
```

Comment:
yeah agreed - i started with email then changed to signup but it does read better and match the intent so I will change this.